src/utils.ts

```diff
@@ -120,6 +120,20 @@ export function objectToQueryString(params: StringMap): string {
     return queryString;
 }
 
+/**
+ * Parse the error message code received from the server during the script load.
+ * For validation errors the response will include a `throw new Error()` statement.
+ * This function execute the received string code.
+ * NOTE: Server response example: throw new Error("detail message");
+ * 
+ * @param source the received error response from the server
+ * @throw {Error} server error message
+ * @returns a string or throw the exception
+ */
+export function parseErrorMessage(source: string): string {
+    return Function(`'use strict'; ${source}`)();
```

Comment:
What happens if someone uses an external URL to load a custom script returning a 400 status with malicious code inside. This line will have an injected script running inside our library or this is not a risk at all.